### PR TITLE
remove use of arcgis gradle plugin

### PR DESCRIPTION
**Pull request by tschie**
handle native dependencies without the ArcGIS Runtime Gradle plugin

### CODE REVIEW

**Review by Rachael-E — COMMENTED** [review]
I've tested this out, and the samples download and run fine. Though to continue using right click to run the samples, as Tyler suggested I added the environment variable into the run configuration. `ARCGISRUNTIMESDKJAVA_100_5_0=build/natives`. This then seems to run fine. @mbcoder is this then behaving as you'd expect?

File: build.gradle [hunk 1]
```diff
@@ -82,8 +81,19 @@ task downloadData {
...
+    configurations.natives.asFileTree.each {
+        from(zipTree(it))
+    }
+    into "build/natives"
+}
+
 run {
+    dependsOn copyNatives
     mainClassName = project.hasProperty("mainClass") ? mainClass : 'com.esri.samples.map.display_map.DisplayMapSample'
+    environment "ARCGISRUNTIMESDKJAVA_100_5_0", "build/natives"
```
Comment (Rachael-E) [inline build.gradle]: @mbcoder and I were wondering if the environment variable set up should come before applying the mainClassName: if there are any timing issues then the main class name might be found before the runtime environment variable is. If timing could be an issue within the run task, then the environment variable set up should come before class name.

**Review by Rachael-E — APPROVED** [review]
@tschie just a query from @mbcoder and myself about ordering in the run task. Otherwise good to merge!

File: build.gradle (hunk 1 above)
Comment (tschie) [reply build.gradle]: The order of these properties does not matter. All task configurations just set things up. Gradle determines the correct order to do everything.

**Pull request merged**